lib/App/Sqitch/Role/RevertDeployCommand.pm

```diff
@@ -174,6 +176,22 @@ around configure => sub {
     return $params;
 };
 
+sub BUILD {
+    my ($self, $args) = @_;
+
+    if ($self->sqitch->strict) {
```

Comment:
Does this not prevent any deploy or revert at all?

Reply:
This will completely block the use of `checkout` and `rebase`, which inherit from this module; but not `deploy` or `revert` since they do not inherit from this module.

Or have I misunderstood your question?

Reply:
Ah, I see, missed the inheritance. It has been a minute since I spent time with this code. :-)